truss/remote/baseten/service.py

```diff
@@ -45,6 +45,10 @@ def predict(
         self,
         model_request_body: Dict,
     ):
+        print(
```

Comment:
I think it would be better to have the prints in the cli.py file. I could imagine a future in which we want to use this in a python client or something like that where the print statements no longer make sense. We can then use `rich.print` which looks a little nicer

Reply:
Makes sense, done! What do we think about this: 
![image](https://github.com/basetenlabs/truss/assets/14193683/10b94779-096d-429a-9ccf-3b7d437bbc04)